### ISSUE TITLE
[Bug]: Wrong namespace/pod labels

### ISSUE THREAD

**Issue by Re4zOon** [netdata/netdata#16544]
### Bug description

Using netdata locally without cloud has some issues (we can't have cloud connection, firewall is very limited).
Some pods are showing up with the wrong namespace:
![image](https://github.com/netdata/netdata/assets/9249934/def9a012-f7eb-4967-97b0-e3f691e36b6a)

### Expected behavior

Namespace should be grabbed from the pod configuration (which is fine of course).

### Steps to reproduce

It wont help, but...
1. Deploy some namespaces
2. Deploy netdata
3. Check the graph

### Installation method

helmchart (kubernetes)

### System info

```shell
Linux sgupamci01m1 6.2.0-1015-aws #15~22.04.1-Ubuntu SMP Fri Oct  6 21:37:24 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux
/etc/lsb-release:DISTRIB_ID=Ubuntu
/etc/lsb-release:DISTRIB_RELEASE=22.04
/etc/lsb-release:DISTRIB_CODENAME=jammy
/etc/lsb-release:DISTRIB_DESCRIPTION="Ubuntu 22.04.3 LTS"
/etc/os-release:PRETTY_NAME="Ubuntu 22.04.3 LTS"
/etc/os-release:NAME="Ubuntu"
/etc/os-release:VERSION_ID="22.04"
/etc/os-release:VERSION="22.04.3 LTS (Jammy Jellyfish)"
/etc/os-release:VERSION_CODENAME=jammy
/etc/os-release:ID=ubuntu
/etc/os-release:ID_LIKE=debian
/etc/os-release:UBUNTU_CODENAME=jammy
```

### Netdata build info

```shell
# As its kuberentes helm chart, I'll just add that:
# helmchart version: 3.7.77
```

**Comment by ilyam8** [comment]
Hey, @Re4zOon. Gathering pods metadata (namespace is part of it) has nothing to do with connection to the Cloud. I think that is the UI bug.

**Comment by Re4zOon** [comment]
Probably its only visible on local dashboards, hence I added the cloud disclaimer

**Comment by ilyam8** [comment]
Fixed in UI v6.75.1
